go/nbs/store.go

```diff
@@ -92,8 +93,21 @@ type LocalStoreFactory struct {
 	maxTables  int
 }
 
+func CheckDir(dir string) error {
+	stat, err := os.Stat(dir)
+	if os.IsNotExist(err) {
+		return fmt.Errorf("Directory does not exist: %s", dir)
```

Comment:
This is mainly because I felt that the original message (`Error:	stat foo: no such file or directory`) wasn't very user friendly (files are not allowed, and `stat` could be confusing without context) - that being said, users are likely devs and the original error can be tested using `os.IsNotExist` when this gets wrapped in a try/catch down the road (as per @aboodman's earlier comments). In light of that, I think it's better to return this error so that some display logic can detect what kind of error this is.